### DISCUSSION TITLE
JSN-SR04T Ultrasonic Sensor Decimals Accuracy

### DISCUSSION THREAD

**Discussion by luisgarcia87** [arendst/Tasmota#18507] [Q&A]
Hi!

I am using the JSN-SR04T with tasmota succesfully.

When I setup the device (a few months back) I think i changed the output of the sensor to show only 1 decimal. I now need the sensor to output 3 decimals but I can't remember how to tell tasmota to show more decimals on the sensor. I've tried searching the command section on the tasmota website but everything displayed there is either for weight, power, current, pressure or any other than a "distance" sensor.

I'm sure it's an easy task I just ran out of ideas or places to search.

Thanks in advance.

<img width="390" alt="Screen Shot 2023-04-25 at 1 57 02 PM" src="https://user-images.githubusercontent.com/561436/234375840-7cabf869-f6ae-4061-a04f-f3057c2a8a37.png">

**Comment by barbudor** [comment]
This is not a Tasmota Issue
You should have opened a discussion

There is no way of changing that. It is hard-coded to 1 digit

**Reply by luisgarcia87** [reply]
Apologies for opening an issue instead of a discussion.

According to Tasmota's configuration for the [HC-SR04 ultrasonic ranging sensor](https://tasmota.github.io/docs/HC-SR04/) (which should be similar to the JSN-SR04T minus the waterproof part) the displayed configuration showed 3 decimals. 

<img width="728" alt="Screen Shot 2023-04-25 at 3 23 38 PM" src="https://user-images.githubusercontent.com/561436/234396117-3b9aa2e4-ad19-489d-acfc-9980475b70b9.png">

**Reply by sfromis** [reply]
Outdated page, someone should update it. Number of decimals **is** now fixed at 1, instead of the really useless 3 decimals of fake digits.

**Reply by luisgarcia87** [reply]
Got it thank you.
